.cross/aarch64-unknown-linux-musl.Dockerfile

```diff
@@ -7,7 +7,6 @@ RUN dpkg --add-architecture arm64 && \
         wget \
         software-properties-common \
         gnupg \
-        libssl-dev:arm64 \
         libsqlite3-dev:arm64 \
```

Comment:
we can remove also `libsqlite3-dev` this one because it's useless being it glibc based